libs/ledgerjs/packages/types-live/src/feature.ts

```diff
@@ -461,6 +462,7 @@ export type Feature_LldRefreshMarketData = Feature<{
 
 export type Feature_CounterValue = DefaultFeature;
 export type Feature_MockFeature = DefaultFeature;
+export type Feature_BuySellUi = DefaultFeature;
```

Comment:
```suggestion
export type Feature_BuySellUI = DefaultFeature;
```

Reply:
Although I agree, I am hesitant to capitalise as the other flags seem to consistently use full camelCase e.g. `llmFeature` etc